### PR TITLE
Update OpenCollar - rlvrelay.lsl

### PR DESCRIPTION
**Pull request by ghost**
remove some unused NULL_KEYs
improve Object Owner / User names - use URI if avatar not in region

### CODE REVIEW

File: LSL/OpenCollar - rlvrelay.lsl [hunk 1]
```diff
@@ -270,6 +270,13 @@ integer Auth(key object, key user)
     return iAuth;
 }
 
+string Name(key id) // return DisplayName + LegacyName or URI if avatar not in region
+{
+    string sName = llGetDisplayName(id)+"("+llKey2Name(id)+")";
+    //if (sName == "()") sName = "secondlife:///app/agent/"+(string)id+"/completename";
+    if (sName == "()") sName = "secondlife:///app/agent/"+(string)id+"/about";
```
Comment (ghost) [inline LSL/OpenCollar - rlvrelay.lsl:277]: 'about' or 'completename' which one is better? I am not sure...

File: LSL/OpenCollar - rlvrelay.lsl [hunk 2]
```diff
@@ -890,6 +901,7 @@ default {
                     key kCurID=llList2String(g_lQueue,1); //GetQObj(0);
                     string sCom = llList2String(g_lQueue,2);  //GetQCom(0));
                     key kUser = NULL_KEY;
+                    key kOwner = llGetOwnerKey(kCurID);
```
Comment (ghost) [inline LSL/OpenCollar - rlvrelay.lsl:904]: use llGetOwnerKey(kCurID) one time, why repeat many times below ?

File: LSL/OpenCollar - rlvrelay.lsl [hunk 3]
```diff
@@ -941,7 +953,7 @@ default {
                         if (!~llListFindList(g_lAvWhiteList, [(string)kUser]))
                         {
                             g_lAvWhiteList+=[(string)kUser];
-                            g_lAvWhiteListNames+=[llGetDisplayName(llGetOwnerKey(kUser))+"("+llKey2Name(llGetOwnerKey(kUser))+")"];
```
Comment (ghost) [inline LSL/OpenCollar - rlvrelay.lsl:944]: llGetOwnerKey(kUser) ? hmmm... senselessly, I think.

**Comment by WendyStarfall** [comment]
Thanks! `"secondlife:///app/agent/"+(string)id+"/about"` is better, all URIs are this way too.

**Pull request merged**